tsne/bh_sne_src/sptree.h

```diff
@@ -136,5 +138,8 @@ struct SPTree<0> {
   enum { no_children = 1 };
 };
 
+#ifdef __GNUC__
 #pragma GCC visibility pop
 #endif
+
+#endif
```

Comment:
This probably should have always been there but before there were other `#if` in here there wasn't anything to be confused about.
```suggestion
#endif  // !defined(SPTREE_H)
```